frontend/src/features/admin-form/settings/SettingsService.ts

```diff
@@ -91,6 +91,14 @@ export const updateFormAuthType: UpdateFormFn<'authType'> = async (
   return updateFormSettings(formId, { authType: newAuthType })
 }
 
+export const updateFormNricMasking: UpdateFormFn<
```

Comment:
/nit would prefer to use `Mask` instead of `Masking` to be terser

Reply:
Got it, I'll make the changes!